### ISSUE TITLE
v5.1.5 breaks coverage whitelist include directory globbing

### ISSUE THREAD

**Issue by vkunz** [sebastianbergmann/phpunit#2052]
hi,

v5.1.5 of phpunit breaks coverage whitelist include globbing

```xml
    <filter>
        <whitelist processUncoveredFilesFromWhitelist="true" addUncoveredFilesFromWhitelist="true">
            <directory suffix=".php">./bundles/*/src</directory>
        </whitelist>
    </filter>
```

git bisect shows commit 55cca0f707e0db4a069473d7394bf2f08df33b83 as the culprit.

An is_dir check is introduces before any wildcard expansion takes place, thus always failing.

thanks
vkunz

**Comment by sebastianbergmann** [comment]
Thanks for bringing this to my attention. I have reverted the partial fix for #2049 and released PHPUnit 5.1.6.

**Comment by vkunz** [comment]
thank you
